src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/NodeConfigParameters.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Stratis.Bitcoin.IntegrationTests.Common.EnvironmentMockUpHelpers
+{
+    public class NodeConfigParameters : Dictionary<string, string>
+    {
+        public void Import(NodeConfigParameters configParameters)
+        {
+            foreach (KeyValuePair<string, string> kv in configParameters)
+            {
+                if (!this.ContainsKey(kv.Key))
+                    this.Add(kv.Key, kv.Value);
+            }
+        }
+
+        public void SetDefaultValueIfUndefined(string key, string value)
+        {
+            if(!this.ContainsKey(key)) this.Add(key, value);
```

Comment:
Generally doing one line `if` is ok but if you have logic its confusing 
If you put the Add in a new line it flows better for the reader.